contribs/gmf/src/controllers/abstract.js

```diff
@@ -123,7 +123,7 @@ gmf.AbstractController = function(config, $scope, $injector) {
 
   /**
    * Languages URL
-   * @type {Object.<string, string>}
+   * @type {Object!}
```

Comment:
The `!` should be used as a prefix.
What is the reason for removing the `<string, string>` typing?